src/main/java/com/contrastsecurity/cassandra/migration/resolver/java/JavaMigrationResolver.java

```diff
@@ -98,6 +98,8 @@ ResolvedMigration extractMigrationInfo(JavaMigration javaMigration) {
         if (javaMigration instanceof MigrationChecksumProvider) {
             MigrationChecksumProvider checksumProvider = (MigrationChecksumProvider) javaMigration;
             checksum = checksumProvider.getChecksum();
+        } else {
+        	checksum = 0;
```

Comment:
Fixed the root cause of Java migration checksum validation error with commit 1ec13c32f3c2391016ad90348bd9b619e43dfe21.

https://github.com/Contrast-Security-OSS/cassandra-migration/commit/1ec13c32f3c2391016ad90348bd9b619e43dfe21